tests/test_routes.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from webapp.app import app
+
+
+class TestRoutes(unittest.TestCase):
+    def setUp(self):
+        """
+        Set up Flask app for testing
+        """
+        app.testing = True
+        self.client = app.test_client()
+
+    def test_homepage_no_session(self):
+        """
+        When given the index URL without session,
+        we should return a 302 status code redirection
+        """
+
+        self.assertEqual(self.client.get("/").status_code, 302)
```

Comment:
@nottrobin looks like this project need SSO authentication. Do you think I should try to create an SSO session in the client and then test to get a 200 status code?

Reply:
No I think that's overkill for now. I think testing for a 302 response from `/` is fine.

Though I don't think it's necessary right now, if you really want to expand on this with better SSO verification, that would actually be awesome. But let's do it in a follow-up PR so we can merge this one now.